bigchaindb/backend/localmongodb/query.py

```diff
@@ -4,10 +4,12 @@
 
 from bigchaindb import backend
 from bigchaindb.backend.exceptions import DuplicateKeyError
+from bigchaindb.common.exceptions import MultipleValidatorOperationError
```

Comment:
It should be plural `MultipleValidatorsOperationError` :pray:

Reply:
I didn't use the plural because the term `ValidatorOperation` is the name for each update request to validator set i.e.
```json
{"validator": {"pub_key":{"type":"ed25519","data":"4E2685D9016126864733225BE00F005515200727FBAB1312FC78C8B76831255A"},
               "power":10},
 "update_id": "validator01"
}
```
The naming is such because each operation can request update to only 1 validator object. So, the plural would be `ValidatorOperations` or `MultipleValidatorOperation`

Reply:
Fair enough!